compiler/package.json

```diff
@@ -15,6 +15,7 @@
     "link": "yarn link",
     "clean": "esy clean",
     "prepare": "esy install",
+    "prebuild": "yarn workspace @grain/cli clean",
```

Comment:
Reaching across this workspace boundary is gross, but since we **only** want this to run if you do `yarn compiler build` (or `yarn workspace @grain/compiler build`), we need to do it this way 😭